docs/_docs/editor/basics.md

```diff
@@ -45,116 +42,112 @@ By default, when you open Nuclide, the Home page appears.
 The Nuclide Home page gives you quick access to common Nuclide tools and features, as well as
 information regarding how to provide feedback.
 
-## Project and File Explorer
+## Project Explorer
 
-The project and file explorer is on the left-hand side of Nuclide. This is where you can open
-projects, navigate through your project to open files in the [editing area](#editing-area),
-create new files and folders, etc.
+The Project Explorer is on the left side of Nuclide and contains two tabs: **File Tree** and **Source Control**. This is where you can open
+projects, navigate through your project to open files in the [Editing Area](#editing-area),
+create new files and folders, view source control information, etc.
```

Comment:
Nit: Could probably link to the source control section we have (Mercurial) here, but we can do that in a different pull request if you want.